### PR TITLE
Add keyboard support for keys WASD and Spacebar

### PR DESCRIPTION
**Pull request by ammunoz**

### CODE REVIEW

File: src/InputManager.h [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 class InputManager {
 
+    const int MAX_PLAYERS = 4; // Maximum number of players in a game
```
Comment (enochtsang) [inline src/InputManager.h]: Usually I find these belong best in anonymous spaces where they're being used. So in the source file:

```
namespace {
    const int MAX_PLAYER = 4;
}
```

File: src/InputManager.cpp [hunk 2]
```diff
@@ -166,6 +181,11 @@ void InputManager::process_input(SDL_Event* event) const {
             std::cout << "[Player " << player_id << "] ";
 
             switch (event->caxis.axis) {
+
+                /**
+                 * LEFT STICK
+                 */
```
Comment (enochtsang) [inline src/InputManager.cpp]: Are such big comments necessary?

`// LEFT STICK`

shoudl suffice right?

Comment (ammunoz) [reply src/InputManager.cpp]: This would come down to preference. Will follow your preference regardless

File: src/InputManager.h (hunk 1 above)
Comment (ammunoz) [reply src/InputManager.h]: Sure. Would prefer to keep these game-wide constants together though once we find the need for something like that

**Pull request merged**